Homework/Marianna/AbstractionInterfaces/2_3.LightSources/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace _2_3.LightSources
+{
+    class Program
+    {
+        static void Main()
+        {
+            List<ILightSource> list = new List<ILightSource>() {
+            new Halogen( 34, 455),
+            new Incandescent(34, 455),
+            new CFL(364, 4655),
+            new LED(364, 4575),
+            new CFL(364, 465),
+            new LED(364, 475)};
```

Comment:
Try moving the curly brackets to their own lines and autoformatting the doc, like so:
```
List<ILightSource> list = new List<ILightSource>()
{
   ...
};
```